src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetHash.cs

```diff
@@ -140,12 +140,22 @@ protected override void ProcessRecord()
                 }
                 catch (FileNotFoundException ex)
                 {
-                    ErrorRecord errorRecord = new ErrorRecord(ex,
+                    ErrorRecord errorRecord = new ErrorRecord(
```

Comment:
Not blocking, but seems a bit repetitive. 🙂 
```suggestion
                    var errorRecord = new ErrorRecord(
```